bindings/capacitorjs/src/definitions.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { EventType, AccountManagerOptions } from './types';
+export interface IotaWalletMobileTypes {
+    initLogger(path: string): Promise<void>;
+    messageHandlerNew(messageOptions: AccountManagerOptions): Promise<{
+        messageHandler: number;
+    }>;
+    sendMessage(messageOptions: {
+        message: string;
+        handler: number;
+    }): Promise<{
+        result: string;
+    }>;
+    listen(options: {
+        eventTypes: EventType[];
+        messageHandler: number;
+    }, callback: (message: {
+        error: {
+            cause: unknown;
+        };
```

Comment:
Shouldn't this be similar to the `Error` type in https://github.com/amadeu2/wallet.rs/blob/33740c11f28aa8f68bc31776bc786e5795f468fe/bindings/nodejs/lib/MessageHandler.ts#L61